app/resources/v1/flags.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+const util = require('util')
+const config = require('config')
+const logger = require('../../../lib/logger.js')()
+
+const readFile = util.promisify(fs.readFile)
+
+async function getFlags (res) {
```

Comment:
Instead of `getFlags` let's rename to `readFlags` to better reflect what this function is doing.